README.md

```diff
@@ -346,7 +346,7 @@ IEEE Int. Conf. Computational Photography (ICCP), Stanford, CA, USA, 2017, pp. 1
 *The Event-Camera Dataset and Simulator: Event-based Data for Pose Estimation, Visual Odometry, and SLAM.*
 - Vasco, V., Glover, A., Mueggler, E., Scaramuzza, D., Natale, L., Bartolozzi, C.  
 *Independent Motion Detection with Event-driven Cameras*,  
-Int. Conf. Advanced Robotics (ICAR), Hong Kong, 2017. [PDF](https://arxiv.org/pdf/1706.08713.pdf)
+Int. Conf. Advanced Robotics (ICAR), Hong Kong, 2017. [PDF](https://arxiv.org/pdf/1706.08713v2.pdf)
```

Comment:
The link in the previous commit (without the "v2" suffix) links to the **latest** version of the paper. See for example [this one](https://arxiv.org/pdf/1610.08336.pdf), which should open v3.
I do not think that it matters for this paper, but it is good to know this trick :)